src/build.py

```diff
@@ -840,6 +858,27 @@ def V8():
     CopyBinaryToArchive(os.path.join(V8_OUT_DIR, a))
 
 
+def Jsc():
+  buildbot.Step('JSC')
+  Mkdir(JSC_OUT_DIR)
+
+  proc.check_call(['xcrun', PREBUILT_CMAKE_BIN, '-Wno-dev',
+                   '..', '-G', 'Ninja',
+                   '-DCMAKE_BUILD_TYPE="Release"',
+                   '-DPORT=Mac',
+                   '-DENABLE_WEBASSEMBLY=ON'],
+                  cwd=JSC_OUT_DIR)
+  proc.check_call(['ninja', 'jsc'], cwd=JSC_OUT_DIR)
+  proc.check_call(['../Tools/Scripts/run-javascriptcore-tests',
```

Comment:
So this part:
```

  if 'GOMA_DIR' in os.environ:
    compiler_launcher = os.path.join(os.environ['GOMA_DIR'], 'gomacc')
    jobs = ['-j', '50']
  else:
    try:
      compiler_launcher = proc.Which('ccache', WORK_DIR)
      command.extend(['-DCMAKE_%s_FLAGS=-Qunused-arguments' %
                      c for c in ['C', 'CXX']])
    except:
      compiler_launcher = None
```
?

Can you test that it works with the cmake + xcrun magic thing?